src/main/java/com/target/studentinfo/exception/ErrorCode.java

```diff
@@ -0,0 +1,14 @@
+package com.target.studentinfo.exception;
+
+public class ErrorCode {
+    // Error codes
+    public static final int NOT_FOUND = 404;
+    public static final int BAD_REQUEST = 400;
```

Comment:
Don't define these values separately. You can use directly refer them from `HttpStatus` like `HttpStatus.BAD_REQUEST.value()` <- This returns integer 400.